server/szurubooru/api/comment_api.py

```diff
@@ -48,7 +48,7 @@ def create_comment(
     text = ctx.get_param_as_string("text")
     post_id = ctx.get_param_as_int("postId")
     post = posts.get_post_by_id(post_id)
-    comment = comments.create_comment(ctx.user, post, text)
+    comment = comments.create_comment(ctx.user if ctx.user.name else None, post, text)
```

Comment:
Don't we need `ctx.user and ctx.user.name` here?

Reply:
No, `ctx.user` is always present even when logged out. See context.py https://github.com/rr-/szurubooru/blob/e3794352f332e0ac09eba30bd5c0ce06337ac76a/server/szurubooru/rest/context.py#L28-L30